src/applications/personalization/profile360/actions/updaters.js

```diff
@@ -148,7 +149,7 @@ export const saveField = {
   updateWorkPhone: updateVet360Field('/profile/telephones', VET360_CONSTANTS.FIELD_NAMES.WORK_PHONE, 'phone'),
   updateTemporaryPhone: updateVet360Field('/profile/telephones', VET360_CONSTANTS.FIELD_NAMES.TEMP_PHONE, 'phone'),
   updateFaxNumber: updateVet360Field('/profile/telephones', VET360_CONSTANTS.FIELD_NAMES.FAX_NUMBER, 'phone'),
-  updateEmailAddress: updateVet360Field('/profile/email', VET360_CONSTANTS.FIELD_NAMES.EMAIL, 'email'),
-  updateMailingAddress: updateVet360Field('/profile/mailing_address', VET360_CONSTANTS.FIELD_NAMES.MAILING_ADDRESS, 'address'),
-  updateResidentialAddress: updateVet360Field('/profile/mailing_address', VET360_CONSTANTS.FIELD_NAMES.RESIDENTIAL_ADDRESS, 'address')
+  updateEmailAddress: updateVet360Field('/profile/email_addresses', VET360_CONSTANTS.FIELD_NAMES.EMAIL, 'email'),
```

Comment:
It seems odd that email addresses is plural here, but I don't know what the endpoint is, so just pointing it out.

Reply:
Thanks! yes looks a little odd but we may be able to support multiple email addresses in the future https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/routes.rb#L181